docs/STYLES.md

```diff
@@ -4,32 +4,41 @@ We use `prettier` for CSS formatting. Use `npm run fix:format` before committing
 
 ## How we go about styles
 
-We decided to move away from styled components to pure SCSS, further we're going to remove the BOM syntax that remains from the signal codebase.
-
-To avoid collisions you MUST NOT create global styles with generic names that aren't a component.
-Also make sure that global module-classes don't have a class name that is already used (search all files with `grep` or a similar tool)
-
-When using generic class names like `date`, `avatar`, `username`, `error` make sure to scope them in a component scss class.
-
-```scss
-.metadata {
-  .date {
-    color: green;
-  }
-  .avatar {
-    background: url('profile.png');
-  }
-  .error {
-    color: red;
-  }
-}
-```
+### CSS modules
+
+- **The goal is to separate every component style-wise as much as possible from each other, so changing the style of a single component doesn't affect anything else**
+- Every CSS file is named `styles.module.css` or `styles.module.scss` when using SCSS
+- It is imported via `import styles from './styles.module.css'`
+- Every CSS module file lives right _next_ to the component
+- Prefer repetition of styles over DRY
+
+### General
+
+- Prefer `padding-top`, `padding-bottom`, `padding-left` etc. over `padding` to avoid bugs through order-critical dependencies between properties, except when you only need `padding` or `border` etc.
+- Try to avoid hacky `!important` as much as possible (goal is to get rid of them altogether at one point)
+- If you can choose, prefer CSS over SCSS, CSS supports nesting and variables and cool functions, like `calc`!
+
+### Naming
+
+- Class names and variable names are always camelCase (`.searchInput`, `--borderRadius`, etc.)
+- The class name should represent the React component it relates to (for example `SearchInput` is represented with `.searchInput`)
+- Sub-classes are usually modifiers of the component and can have short, "local" names (like `.active` or `.warning`)
+
+### Ordering
+
+- Properties are sorted alphabetically
```

Comment:
can we enforce this with linting?

Reply:
but might not always makes sense like:
```
mask: <value>;
-webkit-mask: <value>;
```
but we could also say such polyfills for old property names should be inserted by the compilation process and not manually 🤔

Reply:
> can we enforce this with linting?

Yeah, I've set this up for other projects as well, can dig it up.

Reply:
I've removed this for now, the linting tooling is very heavy